### PR TITLE
feat:landing page

### PR DESCRIPTION
**Pull request by shaksham08**
image
Landing page for the component library

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Approved

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  line-height: 1;
+  font-weight: 400;
+}
+
+.navbar {
+  height: 9.6rem;
+  display: flex;
+  padding: 0 4.8rem;
+  align-items: center;
+  color: #14213d;
```
Comment (Priya-Patel31) [inline styles/main.css]: use variables for these color rather than using directly !!

**Review by Priya-Patel31 — CHANGES_REQUESTED** [review]
Make these changes as they are necessary !!

File: styles/main.css (hunk 1 above)
Comment (shaksham08) [reply styles/main.css]: sure
I have updated this in the latest commit 
https://github.com/shaksham08/Sugar-UI/pull/1/commits/204d601cc4c5b329d3999d44085b8c8370cee154

**Pull request merged**